app/models/permission.rb

```diff
@@ -0,0 +1,8 @@
+class Permission < ApplicationRecord
+  enum :name => [:read, :update, :delete, :order], :_suffix => true
```

Comment:
What happens if we add another permission in here? We have to make sure we add it to the end of the list, right? Because otherwise if we do like `enum :name => [:create, :read, :update, :delete, :order]`, suddenly `:create` is correlated to a `0` in the database, which used to be `:read`, right? Or really, any time we shuffle this list around, could this end up causing issues?

Reply:
@eclarizio that is a legitimate point. I will be rewworking how I implement the enum and modify the confusing way I have the records being created in the migration.

cc @mkanoor